app/routes/items/detail.js

```diff
@@ -87,6 +101,25 @@ export default AuthorizeRoute.extend({
     return Ember.RSVP.all(ids.map(id => this.loadImage(id)));
   },
 
+  // loads package subform detail
+  async preLoadDetail(detail_type, detail_id) {
+    if (detail_type) {
+      return (
+        this.store.peekRecord(
```

Comment:
I think we have a `loadIfAbsent` helper somewhere, would help shorten this function down. I think prettier had its bit of fun 😄

Reply:
loadIfAbsent is present in orders_detail route its not present as a helper function,
I can move it out to somewhere else so that it would be accessible anywhere.

Reply:
Sounds good :)